sunpy/visualization/mapcubeanimator.py

```diff
@@ -33,15 +33,26 @@ class MapCubeAnimator(imageanimator.BaseFuncAnimator):
     colorbar: bool
         Plot colorbar
 
+    plot_function: function
+        A function to call when each frame is plotted, the function must have
```

Comment:
What do you mean by "frame"?  Do you mean 

"A function to call when each Map from the MapCube is plotted"

or does frame have another technical meaning.  Also, I think an example here would be very useful.